sdk/workloads/Azure.ResourceManager.Workloads/tests/SingleServerInstall.json

```diff
@@ -0,0 +1,64 @@
+{
+    "identity": null,
+    "properties.environment": "NonProd",
+    "properties.sapProduct": "S4HANA",
+    "properties": {
+        "configuration": {
+            "configurationType": "Deployment",
+            "appLocation": "eastus",
+            "infrastructureConfiguration": {
+                "deploymentType": "SingleServer",
+                "subnetId": "/subscriptions/49d64d54-e966-4c46-a868-1999802b762c/resourceGroups/SAP-E2ETest-rg/providers/Microsoft.Network/virtualNetworks/loop-test-vnet/subnets/loopsubnet",
+                "virtualMachineConfiguration": {
+                    "vmSize": "Standard_E32ds_v4",
+                    "imageReference": {
+                        "publisher": "RedHat",
+                        "offer": "RHEL-SAP-HA",
+                        "sku": "84sapha-gen2",
+                        "version": "8.4.2021091202",
+                        "exactVersion": null,
+                        "sharedGalleryImageId": null
+                    },
+                    "osProfile": {
+                        "adminUsername": "adminuser",
+                        "adminPassword": null,
+                        "osConfiguration": {
+                            "osType": "Linux",
+                            "disablePasswordAuthentication": true,
+                            "ssh": {
+                                "publicKeys": [
+                                    {
+                                        "keyData": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDPasKQ7pRBFzDsyM3y8UiH+ZvhDeGzqUsuTPktiz/YJeufmgjWnKqYTuZX/tLhSAtX/imG9dBDCLEe5bxLtOn1m2i1vRCfsh2NxNqVxSl2GRXTnsC9Ehw5PHqXQhxSHNItDD1K6vKlydVw7DhmJgkl+1Og7WPdxeO/D6YQr1ogmaQHoK2A0AJyH9HXGbNtIiFUuPers8sKhzYztPSZCO9EZvYTB7quNSa7usgv6CByiJ2MOre/RuLaE7B7lRKhysjeIn1M+fQ3xKxW9cXsJhAHTgbqO2peydz5TLpP0S9XuIna0O5pQEXFgUnfdX/W6NT94Vl/Cqs8obSfQntZ/QmGvr1Qb27ZBSQ24eIsYDfr1mI423TuXwUr+1AEvaSCapgRBnDaV3nBzk9qp1kIIGWRGpaz6qb3ahYpDGP+A8lABy19rBk+/Kmfb9N8qkUK7kdNytHeusKoR0kzhW8xpctCnd/13oN3gXbps0Tx8O2E5w7n+BIDNodxEGxzdYD1hh0= generated-by-azure"
```

Comment:
we should not checkin ssh keys, let's keep a separate empty file where we can add a ssh file on the run and that can be cleared before checkin. add ssh key to this variable dynamically in code after parsing. you can keep a dummy value for now to avoid null check failures while parsing.

Reply:
Keys are moved out.